### PR TITLE
switch to jasmine

### PR DESCRIPTION
**Pull request by trevj**
I forget why we went with Mocha for this repo; since everything else is on Jasmine, I'd strongly prefer we use that.

### CODE REVIEW

File: shadowsocks_config.spec.js [hunk 1]
```diff
@@ -0,0 +1,250 @@
+"use strict";
```
Comment (albertolalama) [inline shadowsocks_config.spec.js]: Do we need to version control this file?

File: .gitignore [hunk 2]
```diff
@@ -1,8 +1,3 @@
-bower_components/
```
Comment (albertolalama) [inline .gitignore]: Is this not useful anymore?

File: shadowsocks_config.spec.js (hunk 1 above)
Comment (trevj) [reply shadowsocks_config.spec.js]: Good catch. Nope, fixed.

File: .gitignore (hunk 2 above)
Comment (trevj) [reply .gitignore]: No. We don't actually call `bower install` anywhere - removing references to Bower may be another cleanup step.

**Pull request merged**